### ISSUE TITLE
[Enhancement] Ability to choose whether to upload multiple images as a set or as individual shots

### ISSUE THREAD

**Issue by EmberHeartshine** [mdshack/shotshare#37]
Title. In order to upload images as individual shots instead of an album, one has to upload them one-by-one. Perhaps if multiple images are detected a dialogue could appear asking the user whether they want to upload as a set or as individual shots.

**Comment by mdshack** [comment]
This is something I have spent a fair amount of time thinking on, I essentially want the upload process to be as short as possible with little to no interaction other than dragging + dropping/pasting etc - I feel like adding additional steps could make this a bit slower/harder to do on the go. One area this can be seen is with stuff like the image name, I didn't really even want that to be a required field/block users from publishing their image. Let me know if you have any thoughts with this in mind!

**Comment by EmberHeartshine** [comment]
You're not wrong, and this is definitely a tough one. I'm not sure there's a great solution while still not adding an extra step. I'll think on it and let you know if I come up with something.

**Comment by EmberHeartshine** [comment]
I made two mockups that might work. The first I'm not 100% on, but if you're able to detect whether multiple images are coming in *before* they're dropped into the field the field could divide itself like this:

![bitmap](https://github.com/mdshack/shotshare/assets/50666308/d35829e2-f46d-4cb9-bc59-0a65daa974ac)

Otherwise, something like this might work:

![bitmap2](https://github.com/mdshack/shotshare/assets/50666308/9667a7f4-526d-4655-ae84-8a350b8c314e)

They're both kind of clunky, but the lower layout would work on mobile where the upper will probably not (or require additional input). That's what I have for now, I'll keep thinking on it.

**Comment by mdshack** [comment]
> I made two mockups that might work. The first I'm not 100% on, but if you're able to detect whether multiple images are coming in _before_ they're dropped into the field the field could divide itself like this:
> 
> ![bitmap](https://private-user-images.githubusercontent.com/50666308/297402005-d35829e2-f46d-4cb9-bc59-0a65daa974ac.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMxNDE2MTEsIm5iZiI6MTcxMzE0MTMxMSwicGF0aCI6Ii81MDY2NjMwOC8yOTc0MDIwMDUtZDM1ODI5ZTItZjQ2ZC00Y2I5LWJjNTktMGE2NWRhYTk3NGFjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE1VDAwMzUxMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTYyMTYxNWJkZWY2N2Q0MWNlMzI1MGZmY2I0ZTU4MzgxYzE3MDBhMWI2NjRkZTI3YjVjMjhmODZlOThkMmViNzQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Uv3XGNTp0WfU16pX4nnRksXrEWy27Cwb8rNfKgx2IJE)
> 
> Otherwise, something like this might work:
> 
> ![bitmap2](https://private-user-images.githubusercontent.com/50666308/297403276-9667a7f4-526d-4655-ae84-8a350b8c314e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMxNDE2MTEsIm5iZiI6MTcxMzE0MTMxMSwicGF0aCI6Ii81MDY2NjMwOC8yOTc0MDMyNzYtOTY2N2E3ZjQtNTI2ZC00NjU1LWFlODQtOGEzNTBiOGMzMTRlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE1VDAwMzUxMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFmMWZkNjJhODlmMGUzOTU0MzAzYTJmYWZiYThhNDhkNDU3ZDE5YmRkOGE0NjdlNmQ5Mjk1ZjAwNTkyOWIzZTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.uYWkTLbFihxc0vYvw6U8MqDLdGLB0GY11kAwoT4m2aQ)
> 
> They're both kind of clunky, but the lower layout would work on mobile where the upper will probably not (or require additional input). That's what I have for now, I'll keep thinking on it.

To achieve the first, I would probably have two dropzones in the same "box", could give something similar to that - Let me ponder this a bit more and see what I can come up with.
